scipy/signal/lti_conversion.py

```diff
@@ -382,7 +386,8 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
 
     The Zero-Order Hold (zoh) method is based on [1]_, the generalized bilinear
     approximation is based on [2]_ and [3]_, the First-Order Hold (foh) method
-    is based on [4]_.
+    is based on [4]_. All conversion methods take place in the state-space form
```

Comment:
I think this sentence might be worded a bit more clearly as "All input are converted to state-space if not already, and then conversion is applied." or something like that.